app/src/main/res/values-h770dp/integers.xml

```diff
@@ -5,7 +5,7 @@
 <resources>
     <!-- Used by the Echo Show 2
```

Comment:
nit: Similarly "Smallest device that uses this file is Echo Show 2, at time of writing."